pyapp/cookiecutter.json

```diff
@@ -14,6 +14,7 @@
     "devdata": ["no", "yes"],
     "postgres": ["no", "yes"],
     "docker": ["no", "yes"],
+    "typecheck": ["no", "yes"],
```

Comment:
If we're removing the `{% if %}` and just adding `make typecheck` and `tox -e typecheck` to all projects, then we can remove this